### ISSUE TITLE
Support for Cuda 12.1

### ISSUE THREAD

**Issue by ammaryasirnaich** [mit-han-lab/torchsparse#329]
I have Cuda 12.1 installed but it seems the framework supports Cuda 11.8, is there any workaround instead of downgrading the Cuda version?

```
The detected CUDA version (12.1) mismatches the version that was used to compile
PyTorch (11.8). Please make sure to use the same CUDA versions.

```

**Comment by zniihgnexy** [comment]
wierd i have the opposite:

The detected CUDA version (11.8) mismatches the version that was used to compile
PyTorch (12.1). Please make sure to use the same CUDA versions.

did you solve it? if so, could you tell me how do you fix that, many thanks :)

**Comment by RFLeijenaar** [comment]
Do you use Conda to install PyTorch by any chance? In that case it might help to remove:
  - cuda-toolkit==12.1 (or any other version).

When you have (in the environment description):
  - pytorch-cuda==12.1
 
It should already install the correct cudatoolkit version. This fixed the issue for me in building another package.

**Comment by Peach-tao622** [comment]
> Do you use Conda to install PyTorch by any chance? In that case it might help to remove:
> 
> * cuda-toolkit==12.1 (or any other version).
> 
> When you have (in the environment description):
> 
> * pytorch-cuda==12.1
> 
> It should already install the correct cudatoolkit version. This fixed the issue for me in building another package.

Yes, your idea is good. But I got other issue when installing this tool,.torchsparse/backend/others/query_cpu.cpp:6:10: fatal error: google/dense_hash_map: No such file or directory
6 | #include <google/dense_hash_map>
  |          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
error: command 'gcc' failed with exit status 1  
My environment setup is as follows:

PyTorch 2.1.0
CUDA 11.8
Python 3.10

If it is convenient for you, could you kindly provide a pre-built wheel package for this environment or a detailed installation guide? Additionally, if there is an updated website where the wheels can be accessed, that would be immensely helpful.
